### DISCUSSION TITLE
我用wix.com做了一個遊戲網站😎

### DISCUSSION THREAD

**Discussion by harrycoconut** [1101-GS4719-C/Python-Community-GS4719-C#1275] [General]
上了資管系半學期的網頁程設課程，我突然想起來我有做過一個關於遊戲的網站，於是我前幾天點開來看發現其實滿猛的，在這邊可以自由的組隊跟討論遊戲。以下是網址，大家可以點進來看看😎
 https://cybermalz.wixsite.com/playtogether

**Comment by tower20021202** [comment]
我進去看了一下，真的好強，希望我在網頁程式課期末結束前，也可以做出跟你一樣厲害的網頁

**Reply by harrycoconut** [reply]
絕對可以的！而且我上課好像做你旁邊，我們一起努力💪

**Reply by tower20021202** [reply]
真的嗎，哈哈

**Comment by Tzuhsuannnnn** [comment]
Oh my god 這東西超酷，希望之後有一天能夠變成大家都在用的網站，讓大家都能透過他找到跟自己合得來的遊戲夥伴!

**Comment by llh26** [comment]
看了一下文章作者設計的網站，我認為網站的內容和排版是很完整的，包括會員註冊、電玩快訊⋯等，都是非常酷且有創意的內容，有機會的話想聽看看作者的設計內容是怎麼樣去構思的，也期許自己在這學期上完網頁設計的課程後，能實作出一個完整的網頁～
